master/buildbot/status/status_gerrit.py

```diff
@@ -349,6 +350,9 @@ def sendCodeReview(self, project, revision, result):
         if message:
             command.append("--message '%s'" % message.replace("'", "\""))
 
+        if self.gerrit_notify:
+            command.append("--notify %s" % self.gerrit_notify)
```

Comment:
Good point.

It should be (as well as the line above), however since it's a parameter to `ssh` command, this will somehow work anyway.

@yurboc I'd appreciate if you fixed this line as well as the one above.